tutor/commands/jobs.py

```diff
@@ -173,6 +173,9 @@ def importdemocourse(
 # Import into CMS
 python ./manage.py cms import ../data "$course_root"
 
+# Work around bug where outline is not generated
+python ./manage.py cms update_course_outline course-v1:OpenedX+DemoX+DemoCourse
```

Comment:
Is it impacting all the courses or only under certain circumstances? If the outline is not generated in initial import, does it not generate when a user navigates to LMS  or edits something on CMS?

Reply:
@arbrandes Hi, catching up on ☝🏽

Reply:
@arbrandes Hi, pinging for ☝🏽 . Thanks

Reply:
That's a good question.  I would guess this impacts all courses that are imported using the same command, yes, but I haven't actually tested it.

Reply:
@arbrandes Were you able to test ☝🏽 ? Thanks

Reply:
If it happens for every course, how does this fix the larger problem?